From 1ea825f33166b0bbd0c1763d83d3932f37867a3d Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Sat, 28 Mar 2020 11:10:17 -0400
Subject: [PATCH 1481/1936] CI: Add apt-get update step

This commit should resolve some issues when available versions of packages
change over time (currently causing the first apt-get install command to fail).
---
 .github/workflows/ccpp.yml | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/.github/workflows/ccpp.yml b/.github/workflows/ccpp.yml
index eecd0d12..1ac76764 100644
--- a/.github/workflows/ccpp.yml
+++ b/.github/workflows/ccpp.yml
@@ -6,9 +6,11 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
-    
+
     steps:
     - uses: actions/checkout@v1
+    - name: update_apt
+      run:   sudo apt-get update
     - name: install_deps1
       run:   sudo apt-get install zlib1g-dev libmxml-dev libfftw3-dev dssi-dev libfltk1.3-dev fluid libxpm-dev
     - name: install_deps2
-- 
2.47.0

